Data/Sys/GameSettings/RODE01.ini

```diff
@@ -25,6 +25,5 @@ PH_ExtraParam = 0
 PH_ZNear =
 PH_ZFar =
 
-[Video_Hacks]
-EFBEmulateFormatChanges = True
-
+[Video_Settings]
+AspectRatio = 2
```

Comment:
Well, as far as I know, it's the only Wii game that supports 4:3 only so I still think the AR should be preset in this game given the default Dolphin settings would result in stretched graphics but I'm the minority here, will remove that...